src/common/browser-info/browser-info.ts

```diff
@@ -4,5 +4,6 @@ export default interface BrowserInfo {
     language: string;
     screen_height: number;
     screen_width: number;
+    time_zone?: string;
```

Comment:
I know `time_zone` is only required for a particular case, but since it is pretty cheap to send it, I would argue we should send it all the time.

In fact, assuming we can deduce `time_zone_offset` from `time_zone` we could just replace one with the other and when we need `time_zone_offset` just use `time_zone` in the backend to generate it.